frontend/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
     <link rel="stylesheet" href="/assets/css/main.css">
+    <link rel="manifest" href="/assets/pwa-manifest.json" />
```

Comment:
Nitpicking, but I believe the customary extension is `.webmanifest`, so that the right media type `application/manifest+json` can be applied.

Reply:
good to know!